salt/modules/win_pki.py

```diff
@@ -167,6 +167,8 @@ def get_cert_file(name, cert_format=_DEFAULT_FORMAT):
     :param str name: The filesystem path of the certificate file.
     :param str cert_format: The certificate format. Specify 'cer' for X.509, or
         'pfx' for PKCS #12.
+    :param str password: The password of the certificate. Only applicable to pfx
+        format.
```

Comment:
Could we add a warning in the documentation stating that this password will be seen by all minions? See the discussion below on this.